io.github.iSlammedMyKindle.godotGem.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.iSlammedMyKindle.godotGem
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: godotGemClient.x86_64
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
Godot (the engine that this project is using) doesn't support wayland yet. It looks like regardless if I use this flag or not, it's running on X11. I'll remove this flag for you